source/styleEditor.js

```diff
@@ -1,3 +1,12 @@
 $(document).ready(function(){
-  // your code goes here.
+  var selector;
```

Comment:
You could probably just assign these at the same time in the body of the function 👍